proto/zetachain/zetacore/authority/tx.proto

```diff
@@ -53,8 +55,17 @@ message MsgUpdatePoliciesResponse {}
 // MsgUpdateChainInfo defines the MsgUpdateChainInfo service.
 message MsgUpdateChainInfo {
   string creator = 1;
-  ChainInfo chain_info = 2 [ (gogoproto.nullable) = false ];
+  pkg.chains.Chain chain = 2 [ (gogoproto.nullable) = false ];
```

Comment:
Will this change break backwards compatibility?

Reply:
Yes, after this release, you need to submit transactions with `Chain` and not `ChainInfo.` 
I am not sure what you mean by compatibility; the chain info object stored on the state would still be the same

Reply:
Basically will a similar issue to this happen: https://github.com/zeta-chain/node/issues/2835

Reply:
No, this changes a field in the transaction.
I added a comment to track this though , as I am still not very sure what is the root cause of original issue

Reply:
I agree with Alex. It should not be possible to change **existing** proto fields. It should be `reserved 2` + `pkg.chains.Chain chain = 3`

Reply:
Both changes have already been done 
https://github.com/zeta-chain/zeta-node/blob/89db4b85296a2b32fc13d17b07d0f51650298de4/proto/zetachain/zetacore/authority/tx.proto#L57-L58